docs/docs-source/docs/modules/get-started/pages/install-cloudflow.adoc

```diff
@@ -0,0 +1,322 @@
+--namespace cloudflow --set operatorImageName="lightbend/flinkk8soperator" --set \
+operatorVersion="v0.5.0"
+----
+
+This completes the installation of the Flink operator. 
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS      RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running     0          3m29s
+cloudflow-patch-spark-mutatingwebhookconfig-66xxb   0/1     Completed   0          21s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running     0          2m29s
+flink-operator-7999fdd879-phqlg                     1/1     Running     0          1m29s
+----
+
+== Installing Enterprise components
+If you have a commercial agreement with Lightbend, you are eligible to install the Cloudflow Enterprise components. 
+
+Before installing, you need to make sure you have the credentials required to download the Helm chart from the Lightbend commercial repository.
+
+Add the Lightbend Helm repository.
+
+  helm repo add lightbend https://repo.lightbend.com/helm-charts/
+
+Update Helm to make sure all charts are up-to-date
+
+  helm repo update
+
+Install the components, note that you set the `$USERNAME` and `$PASSWORD` environment variables before executing this command. 
+
+We recommend using the following command to create an environment variable for username and password. 
+*This is to avoid storing this information in the shell's history file.*
+
+  read -a USERNAME
+  read -a PASSWORD
+
+Once configured, we can execute the Helm command to install the Cloudflow enterprise components. Note how we use the `$USERNAME` and `$PASSWORD` in the Helm command.
+ 
+----
+helm upgrade cloudflow-enterprise-components lightbend/cloudflow-enterprise-components \
+  --install \
+  --namespace cloudflow \
+  --set enterpriseOperatorVersion=2.0.8 \
+  --set enterprise-suite.imageCredentials.username="$USERNAME" \
+  --set enterprise-suite.imageCredentials.password="$PASSWORD"
+----
+
+After we have used the username and password environment variables, we can clear them by closing the shell or executing the following commands:
+
+  unset USERNAME
+  unset PASSWORD
+
+To quickly verify that the installation worked, you can open a proxy to the Lightbend console service and confirm that you can access the Lightbend console UI.
+
+First, create the port-forward.
+
+  kubectl port-forward -n cloudflow svc/console-server 5000:80
+
+Then open the following URL in your browser:
+
+  https://localhost:5000
+
+You should now see the Lightbend console in your browser.
+
+== Upgrading Cloudflow
+
+If you need to upgrade Cloudflow to a newer version, the following chapter in the administration section will show you how.
+
+. xref::administration:upgrading-cloudflow.adoc[upgrading Cloudflow].
+
+== What's next
+
+Now, we are ready to xref:deploy-to-gke-cluster.adoc[deploy an example application to the cluster].
```

Comment:
I forgot to install NFS before everything and then caused issues. I had to delete the namespace of the app, so the PVC would actually work, after installing NFS. Will have to look into this.

Reply:
Interesting, as an experiment I deployed an Akka app without installing the NFS controller, so the PVC was left pending, once I installed the NFS controller, the PVC was created as normal. So that problem is likely caused by any of the pods expecting the volume being there?